packages/material-ui/src/Badge/Badge.js

```diff
@@ -56,7 +56,7 @@ const overridesResolver = (props, styles) => {
 const BadgeRoot = styled(
   'span',
   {},
-  { muiName: 'MuiBadge', overridesResolver },
+  { name: 'Badge', slot: 'Root', overridesResolver },
```

Comment:
I've been meaning to ask – is "slot" a recognised term? I hadn't come across it before your use, and a quick web search doesn't bring up anything relevant.

Reply:
It’s not something standardized, I’ve just got used to use it for describing different parts of the component. In Vue, slots are used as templates part for creating a layout https://vuejs.org/v2/guide/components-slots.html Here is an example:
```
<div class="container">
  <header>
    <!-- We want header content here -->
    <slot name="header"></slot>
  </header>
  <main>
    <!-- We want main content here -->
    <slot name="body"></slot>
  </main>
  <footer>
    <!-- We want footer content here -->
    <slot name="footer"></slot>
  </footer>
</div>
```

I don’t think react has a similar notion for it, but I think we can just use the same term. Open for other options around naming here :)

Reply:
I was influenced by the usage of the slot terminology by [Vue](https://vuejs.org/v2/guide/components-slots.html) and [Web Components](https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_templates_and_slots) when making the initial proposal. The first idea that came to mind was `component` but it seemed to be overloaded. Hence slot as my second option. 

Any other idea?

Reply:
[Part](https://chakra-ui.com/docs/theming/component-style#styling-multipart-components)

Reply:
`part` is the other term I had in mind, but `slot` sound better to me 🤷‍♀️

Reply:
svelte has slots as well.

Reply:
I'm struggling to see how "slot" as used here relates to slots in Vue and Svelte, where it appears to be a placeholder for children equivalent to `{props.children}` in React. A slot is something that you put things into. That doesn't seem to be the case here, unless I'm misunderstanding.

Reply:
@mbrookes each `components`' prop key is a slot.

Reply:
Okay, thanks.